src/App.tsx

```diff
@@ -1,17 +1,64 @@
 import InputBox from './components/InputBox';
 import Task from './components/Task';
+import React, { useState } from 'react';
+import { DragDropContext, DropResult } from 'react-beautiful-dnd';
 
 export default function App() {
+  const [todo, setTodo] = useState <string> ("");
+  const [todos, setTodos] = useState <string[]> ([]);
+  const [completedTodos, setCompletedTodos] = useState <string[]> ([]);
+
+  const handleAdd = (event: React.FormEvent) => {
+    event.preventDefault();
+    if (todo != "") {
```

Comment:
```suggestion
    if (todo != "") {
```
Will `!==`  a better option? P.S. Im not sure as I haven't read meticulously